src/app/[locale]/booking/booking.module.css

```diff
@@ -30,4 +30,8 @@
   .stepper{
     height:100vh;
   }
+<<<<<<< HEAD
```

Comment:
It looks like a leaked not resolved conflict

Reply:
Cleaned up